docs/how_to_make.md

```diff
@@ -0,0 +1,87 @@
+# How to make a WatchDuino 2
+
+## Bill of materials
+
+You'll need the following in order to build a WatchDuino 2:
+
+- 1 × Arduino mini pro 2 8Mhz 3.3v
+- 1 × LIPO battery 240mah
+- 1 × 0.96” I2C 128X64 OLED
+- 4 × 3 x 6 x 4.3mm Push Button 2 Pin
+- 1 × 0.06A Micro Vibrator Motor
+- 1 × Micro USB LIPO charger External charger
+- 1 × Bluetooth 4.0 BLE module HM-10
+
+## Loading the WatchDuino firmware in the Arduino Mini
+
+Before you even start building, it is advisable to load the WatchDuino
+firmware into the Arduino Mini. You can also do this at the end, but if you do
+it now you'll be able to try the components as you go along.
+
+These are the steps you need to follow:
+
+1. To install the firmware on a device you'll need the Arduino SDK and IDE.
+You can download the latest bundle for your platform
+[on the official website](https://www.arduino.cc/en/Main/Software),
+for free. Unzip the package to any location of your choice. Remember this
+location, because this will be the `Arduino SDK`, everytime we refer to it.
+Open the `arduino` executable inside the SDK folder to run the IDE.
+
+2. Copy all the folders inside the `libraries` folder in this repo into the
+`libraries` folder of your Arduino SDK. These are dependencies the WatchDuino
+code needs to compile.
+
+3. On the Arduino IDE, click on `File->Open`. Navigate to where you cloned
+the WatchDuino repository, and pick the `watch/watch.ino` file to open.
+All of the WatchDuino files should now have been loaded in the IDE.
+
+4. On the IDE now go to `Tools->Board` and pick `Arduino Pro or Pro Mini`.
+Then pick `Processor:ATMega328 (3.3V, 8Mhz)`. This step is needed because the
+IDE needs to know the microcontroller that we are using, in order to apply
+the correct compilation options.
+
+5. After all this, you should be able to compile the code. On the IDE, click
+on the `Verify` button (the first icon in the toolbar, showing a tick). After a
+minute it should be done without errors.
+
+6. You are ready now to upload the code into your Arduino Mini. Connect
```

Comment:
Arduino mini requires TTL module (comes without built-in usb interface) https://www.arduino.cc/en/pmwiki.php?n=Guide/ArduinoProMini